wolfssl/internal.h

```diff
@@ -2622,6 +2628,20 @@ typedef struct DtlsMsg {
 
 #endif
 
+#ifdef HAVE_LWIP_NATIVE
+    /* LwIP native tpc socket context */
+    typedef struct LwIP_native_Ctx {
+        struct tcp_pcb * pcb ;
+        tcp_recv_fn recv ;
+        tcp_sent_fn sent ;
+        int    pulled  ; 
+        struct pbuf *pbuf ;
+        int    wait ;
+        void * arg ;   /* arg for application */
+        int    idle_count ;
+    } LwIP_native_Ctx ;    
```

Comment:
Please remove spaces before ;. Typo above with "tpc". Please use a name to indicate this is a wolf structure (like WOLF_LWIP_NATIVE_CTX).